hyperscale/Dockerfile

```diff
@@ -0,0 +1,73 @@
+      org.label-schema.name="Citus" \
+      org.label-schema.description="Scalable PostgreSQL for multi-tenant and real-time workloads" \
+      org.label-schema.url="https://www.citusdata.com" \
+      org.label-schema.vcs-url="https://github.com/citusdata/citus" \
+      org.label-schema.vendor="Citus Data, Inc." \
+      org.label-schema.version=${VERSION} \
+      org.label-schema.schema-version="1.0"
+
+ENV CITUS_VERSION ${VERSION}.citus-1
+ENV POSTGRES_DB citus
+
+# install Citus and needed extension on hyperscale
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+       ca-certificates \
+       curl \
+    && curl -s https://install.citusdata.com/community/deb.sh | bash \
+    && apt-get install -y postgresql-$PG_MAJOR-citus-10.2=$CITUS_VERSION \
+                          postgresql-$PG_MAJOR-hll=2.16.citus-1 \
+                          postgresql-$PG_MAJOR-topn=2.4.0 \
+                          postgresql-14-postgis-3 \
+                          postgresql-plperl-14 \
+                          postgresql-contrib \
+                          postgresql-14-partman \
+                          postgresql-14-tdigest \
+                          postgresql-14-mysql-fdw \
+                          postgresql-14-cron \
+    && apt-get purge -y --auto-remove curl \
+    && rm -rf /var/lib/apt/lists/*
+
+
+
+# set shared_preload_libraries to the list of extensions that needs to be loaded when postgresql starts
+RUN echo "shared_preload_libraries='citus,pg_cron,pg_stat_statements'" >> /usr/share/postgresql/postgresql.conf.sample
+
+# set default database name for pg_cron extension
+RUN echo "cron.database_name = 'citus'" >> /usr/share/postgresql/postgresql.conf.sample
+
+# add scripts to run after initdb
+COPY hyperscale/001-db-user-creation.sql /docker-entrypoint-initdb.d/
+COPY hyperscale/002-create_extension_functions.sql /docker-entrypoint-initdb.d/
+COPY hyperscale/003-create_citus_extension.sql /docker-entrypoint-initdb.d/
+COPY hyperscale/004-postgres_replication_origins.sql /docker-entrypoint-initdb.d/
+COPY hyperscale/005-create_postgis_extension.sql /docker-entrypoint-initdb.d/
+COPY hyperscale/006-other_extensions.sql /docker-entrypoint-initdb.d/
+COPY hyperscale/007-pg_cron_pg_partman_setup.sql /docker-entrypoint-initdb.d/
+COPY hyperscale/008-grant_usage_fdw.sql /docker-entrypoint-initdb.d/
+COPY hyperscale/009-other_grants.sql /docker-entrypoint-initdb.d/
+COPY hyperscale/010-stat_statements_reset.sql /docker-entrypoint-initdb.d/
+
+# add health check script
+COPY pg_healthcheck wait-for-manager.sh /
+
+RUN chmod +x /wait-for-manager.sh
+
+# This block is required to be able to define the password for user citus. With the below script citus password can be
+# set using 'CITUS_PASSWORD' env variable set in docker run phase
+RUN apt-get update && apt-get install -y  sudo && adduser postgres sudo
+RUN echo 'postgres     ALL=(ALL) NOPASSWD:ALL' >> '/etc/sudoers'
```

Comment:
I do not think that it is wise to add postgres user as a sudoer. Are you sure this is safe to do?

Reply:
I think this is for development purposes not for production. Therefore, there is no harm for postgres user to have sudo permission. Since it is solely for development purposes, it may make our life easier for the further developments. 
@marcocitus what do you think?